pages/styles/teams.css

```diff
@@ -0,0 +1,24 @@
+.boardMembers {
+  align-items: flex-start;
```

Comment:
It looks like most of these rules get overridden at the bottom of this file.

flex-wrap is the only attribute here that actually applies to the DOM